website/content/docs/v0.14/Learn More/developing-talos.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Developing Talos"
+weight: 13
+---
+
+This guide outlines steps and tricks to develop Talos operating systems and related components.
+The guide assumes Linux operating system on the development host.
+Some steps might work under Mac OS X, but using Linux is highly advised.
+
+## Prepare
+
+Check out the [Talos repository](https://github.com/talos-systems/talos).
+
+Try running `make help` to see available `make` commands.
+You need Docker and `buildx` installed on the host.
+
+> Note: On Ubuntu 21.10 it's easier to use Ubuntu's `docker`, for earlier versions of Ubuntu use [Docker's apt repositories](https://docs.docker.com/engine/install/ubuntu/).
```

Comment:
let's remove this

Reply:
I updated that to use different wording, but I think it still makes sense to use Docker from upstream.